Elevel/Application/Features/ReportCommands/GetReportDetailQuery.cs

```diff
@@ -34,7 +35,15 @@ public Handler(IApplicationDbContext context, IMapper mapper, UserManager<User>
 
             public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
             {
-                var report = await _context.Reports.AsNoTracking()
+                if (request == null)
```

Comment:
How do you can achieve null request object?

Reply:
Deleted it